### ISSUE TITLE
CompositionDrawer does not preserve transformations when writing to SVG

### ISSUE THREAD

**Issue by sbauer322** [openrndr/openrndr#65]
I don't know if this counts as a bug or just my own ignorance, but I'm attempting to use the `CompositionDrawer` with transformations such as `translate`, `rotate`, and `scale`. I'm able to use these successfully with the normal `Drawer`. If it is a bug then I will reformat the issue to match the template.

My problem is that none of the transformations used with`CompositionDrawer` seem to make it into an SVG file. 

Here's a sample code snippet that produces a handful of rectangles all in their original orientation with no rotation when written to an SVG file:

```
fun writeSvgFile(width: Int, height: Int) {
        val symmetry: Int = 6
        val drawer = CompositionDrawer()

        drawer.translate(width / 2.0, height / 2.0)
        val angle = 360.0 / symmetry.toDouble()
        for (i in (0 until symmetry)) {
            drawer.rotate(angle)
            drawer.rectangle(5.0*i, 5.0*i, 10.0, 15.0)
        }

        val composition: Composition = drawer.composition
        val svgDoc = writeSVG(composition)
        File("screenshots/output.svg").writeText(svgDoc)
}
```

It is certainly possible I am misunderstanding something about `CompositionDrawer` in relation to `Drawer` but I haven't seen anything to indicate as such. Below is a slightly styled version of how `Drawer` draws the rectangles.

![Screen Shot 2020-02-02 at 4 04 32 PM](https://user-images.githubusercontent.com/2945386/73615419-b7aadc00-45d5-11ea-8c40-96e204b233c2.png)

**Comment by edwinRNDR** [comment]
@sbauer322 thank you for detailed issue report. I can confirm this is definitely a bug in SVGWriter. SVGWriter appears to omit the transforms from the `Composition` to SVG conversion.  I pushed a quick-fix 2b69f8dd818e36106bf76862fff171dc2ae73e3a in which I added the transform output. 

- [ ] I still have to verify that this works well when used in combination group transforms. 

We will very likely have a release candidate for 0.3.39 up tomorrow in case you want to try it but don't feel like building the library yourself.
